rust/src/config/ja2_json.rs

```diff
@@ -36,17 +37,19 @@ fn build_json_config_location(stracciatella_home: &PathBuf) -> PathBuf {
 }
 
 impl Ja2Json {
+    /// Construct a Ja2Json instance from the stracciatella home directory
     pub fn from_stracciatella_home(stracciatella_home: &PathBuf) -> Self {
         let path = build_json_config_location(stracciatella_home);
         Ja2Json { path }
     }
 
-    pub fn get_content(&self) -> Result<Ja2JsonContent, String> {
+    fn get_content(&self) -> Result<Ja2JsonContent, String> {
```

Comment:
was this change intentional?

Reply:
Yes, the function is not publicly used, so I decided to make it not public. Should have added that to a separate commit.